tsdb/index/tsi1/partition.go

```diff
@@ -340,13 +351,13 @@ func (p *Partition) buildSeriesSet() error {
 
 // CurrentCompactionN returns the number of compactions currently running.
 func (p *Partition) CurrentCompactionN() int {
-	p.Mu.RLock()
-	defer p.Mu.RUnlock()
+	p.mu.RLock()
+	defer p.mu.RUnlock()
 	return p.currentCompactionN
 }
 
 // Wait will block until all compactions are finished.
-// Must only be called while they are disabled.
+// must only be called while they are disabled.
```

Comment:
I think some search and replace for `mu` de-capitalized this `must`.

Reply:
@gwossum --
@davidby-influx suggested some changes in https://github.com/influxdata/influxdb/pull/25338 so I'll make those changes and do `must` -> `Must`.